### DISCUSSION TITLE
단순 오탈자 정리

### DISCUSSION THREAD

**Discussion by tc-ha** [fsoftwareengineer/todo-application#34] [자신의 진행 상황 공유]
p.100 위에서부터 3번째 줄:         HTTP 리퀘스                           ->    HTTP 리퀘스트
p. 111 그림 2-30:  [갈색 박스]     testRequestParam                ->    testRequestBody
p. 200 위에서부터 5번째 줄:        + 버튼 클릭 시                          ->    휴지통 버튼 클릭 시
p. 219 위에서부터 9번째 줄:        Rejecte                                   ->     Reject
p. 225 위에서부터 5번째 줄:        Servce API                             ->     Service API
p. 245 아래에서부터 8번째 줄:    .email(user.getUsername())  ->    .email(user.getEmail())
p. 298 위에서부터 3번째 줄:       localhost:5000/todo              ->    localhost:8080/todo
p. 384 맨 아랫줄:                       플로이하려면                             ->    디플로이하려면

**Comment by epson220** [comment]
p.86 소스코드 수정 
   
 for(Todo todo : todos) {
        JSONObject todoJson = new JSONObject();
        todoJson.put("id", todo.getId());
        todoJson.put("title", todo.getTitle());
        todoJson.put("isdone", todo.isDone());
        array.put(todoJson):
    }

**Reply by fsoftwareengineer** [reply]
오탈자 정리 감사합니다.

**Comment by epson220** [comment]
p. 89 오탈자
// 서비스 레이어
List todos = service.getTodos(request.userId);
->
List todos = todoService.getTodos(request.userId);

**Comment by epson220** [comment]
p.99
레이어 시스템에서 
예를 들어 "서버"가 인증 서버, 캐싱 서버, ......
=>
예를 들어 "요청"이 인증 서버, 캐싱 서버, ....
